routes.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express')
+const application = express()
+const port = 8000
+
+const rampwalk =[
+            {
+              "index": 1,
+              "Event": "Event 1",
+              "Likes": 10,
+              "Rating": 3.2,
+              "Views": "1k"
+            }
+   
+]
+
+
+application.use(express.json())
+
+application.get('/', (req, res) => {
+  res.send(rampwalk)
+})
+
+application.post('/', (req, res) => {
```

Comment:
It's a good practice to validate the incoming request body in your POST and PUT methods. This will ensure that only valid data is added or updated. [important]